packages/lesswrong/server/resolvers/postResolvers.ts

```diff
@@ -1,39 +1,52 @@
-import { googleVertexApi } from '../google-vertex/client';
-import { userCanDo, userIsAdmin } from '../../lib/vulcan-users/permissions';
+import {Posts} from '../../lib/collections/posts/collection'
+import {sideCommentAlwaysExcludeKarma, sideCommentFilterMinKarma} from '../../lib/collections/posts/constants'
+import {Comments} from '../../lib/collections/comments/collection'
+import {
+  getLastReadStatus,
+  sideCommentCacheVersion,
+  SideCommentsResolverResult,
+} from '../../lib/collections/posts/schema'
+import {accessFilterMultiple, augmentFieldsDict, denormalizedField} from '../../lib/utils/schemaUtils'
+import {getLocalTime} from '../mapsUtils'
+import {canUserEditPostMetadata, extractGoogleDocId, isNotHostedHere} from '../../lib/collections/posts/helpers'
+import {matchSideComments} from '../sideComments'
+import {captureException} from '@sentry/core'
+import {getToCforPost} from '../tableOfContents'
+import {getDefaultViewSelector} from '../../lib/utils/viewUtils'
+import keyBy from 'lodash/keyBy'
+import GraphQLJSON from 'graphql-type-json'
+import {addGraphQLMutation, addGraphQLQuery, addGraphQLResolvers, addGraphQLSchema, createMutator} from '../vulcan-lib'
+import {postIsCriticism} from '../languageModels/criticismTipsBot'
+import {createPaginatedResolver} from './paginatedResolver'
+import {
+  getDefaultPostLocationFields,
+  getDialogueMessageTimestamps,
+  getDialogueResponseIds,
+  getPostHTML,
+} from '../posts/utils'
+import {buildRevision} from '../editor/make_editable_callbacks'
+import {cheerioParse} from '../utils/htmlUtil'
+import {isDialogueParticipant} from '../../components/posts/PostsPage/PostsPage'
+import {marketInfoLoader} from '../../lib/collections/posts/annualReviewMarkets'
+import {getWithCustomLoader} from '../../lib/loaders'
+import {isAF, isLWorAF, twitterBotKarmaThresholdSetting} from '../../lib/instanceSettings'
+import {hasSideComments} from '../../lib/betas'
+import SideCommentCaches from '../../lib/collections/sideCommentCaches/collection'
+import {drive} from '@googleapis/drive'
+import {convertImportedGoogleDoc} from '../editor/conversionUtils'
+import Revisions from '../../lib/collections/revisions/collection'
+import {randomId} from '../../lib/random'
+import {getLatestRev, getNextVersion, htmlToChangeMetrics} from '../editor/utils'
+import {canAccessGoogleDoc, getGoogleDocImportOAuthClient} from '../posts/googleDocImport'
+import {getLatestContentsRevision, GoogleDocMetadata} from '../../lib/collections/revisions/helpers'
+import {recombeeApi, recombeeRequestHelpers, RecommendedPost} from '../recombee/client'
+import {
+  HybridRecombeeConfiguration,
+  RecombeeRecommendationArgs,
+} from '../../lib/collections/users/recommendationSettings'
+import {googleVertexApi} from '../google-vertex/client'
+import {userCanDo, userIsAdmin} from '../../lib/vulcan-users/permissions'
+import {FilterPostsForReview} from '@/components/bookmarks/ReadHistoryTab'
```

Comment:
would prefer reverting changes made an autoformatter here, to better preserve git blame

Reply:
I guess to actually achieve this I need to force push the amended commit 🤔